lib/presentation/menu/menu_screen.dart

```diff
@@ -0,0 +1,248 @@
+                            mainAxisAlignment: MainAxisAlignment.center,
+                            children: [
+                              Icon(
+                                Icons.logout,
+                                color: kErrorColor,
+                              ),
+                              SizedBox(
+                                width: 2.5,
+                              ),
+                              Text(
+                                "Log out",
+                                style: TextStyle(
+                                  color: kErrorColor,
+                                  fontSize: 15,
+                                  fontWeight: FontWeight.w700,
+                                ),
+                              )
+                            ],
+                          ),
+                        ),
+                      ],
+                    ),
+                  ),
+                  _buildSubTitle(
+                    subTitle: "Account",
+                    paddingTop: 38.5,
+                    paddingBottom: 20,
+                  ),
+                  Padding(
+                    padding: const EdgeInsets.only(left: 22.25),
+                    child: GestureDetector(
+                      onTap: () {},
+                      child: Row(
+                        crossAxisAlignment: CrossAxisAlignment.start,
+                        children: [
+                          Icon(
+                            Icons.account_circle,
+                            color: kPrimaryColor300,
+                            size: 19.5,
+                          ),
+                          SizedBox(
+                            width: 12.25,
+                          ),
+                          Column(
+                            crossAxisAlignment: CrossAxisAlignment.start,
+                            children: [
+                              Text(
+                                "Account Information",
+                                style: TextStyle(
+                                    fontSize: 12,
+                                    fontWeight: FontWeight.w500,
+                                    color: kPrimaryColor300),
+                              ),
+                              SizedBox(
+                                height: 5,
+                              ),
+                              SizedBox(
+                                width: 306,
+                                child: Text(
+                                  "View and update your account  and contact information that is associated with your CollAction account.",
+                                  style: TextStyle(
+                                      fontSize: 12,
+                                      color: kPrimaryColor300,
+                                      fontWeight: FontWeight.w400),
+                                ),
+                              ),
```

Comment:
Why are you using a fixed width here (why even a sizedbox?)? This won't make it look well on devices.

Reply:
I will fix it soon, sorry I got exams 😥.